### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by avadhut-the-merciful**
I have added Schema.org markup in the head element of the website to enhance its search engine optimization (SEO) and improve how search engines understand and display the website's content. By incorporating Schema.org, I aim to provide a clearer representation of the website’s content to search engines, potentially leading to better rankings and increased visibility.

Information Sources
    URL: "https://activitywatch.net/"
    GitHub: https://github.com/ActivityWatch

Schema Type
I have used "Software Application" as the application type in the schema. Content Details

For descriptions, abstracts, about, and keywords, I referenced the official documentation. Additionally, I included keywords like "ADHD" and "freelancing" to help expand the app's reach.

Contributors
For funders, sponsors, and contributors, I used information from the GitHub page mentioned above and will update it as needed.

Software Requirements
As a placeholder, I have listed the software requirements in schema as: "softwareRequirements": "Requires Windows 10 or later, macOS 10.15 or later, Linux kernel 5.4 or later, or Android 7.0 or later",

Validation
I have checked the code using the Schema.org (https://validator.schema.org/) Validator for any errors.

Please let me know if there are any corrections needed. Thank you.

----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 78742a068d6fe6fe4141b202da422520661a4d69  | 
|--------|--------|

### Summary:
Added Schema.org `SoftwareApplication` markup to `index.html` to enhance SEO and search engine understanding.

**Key points**:
- Added Schema.org `SoftwareApplication` markup to `index.html`.
- Included details such as `name`, `url`, `description`, `keywords`, `abstract`, `alternativeHeadline`, `about`, `applicationCategory`, `applicationSubCategory`, `aggregateRating`, `softwareVersion`, `datePublished`, `dateCreated`, `downloadUrl`, `softwareHelp`, `screenshot`, `fileSize`, `inLanguage`, `featureList`, `audience`, `typicalAgeRange`, `operatingSystem`, `availableOnDevice`, `potentialAction`, `softwareRequirements`, `creator`, `maintainer`, `contributor`, `funding`, `sponsor`, `usageInfo`, `discussionUrl`, `releaseNotes`, and `license`.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by avadhut-the-merciful** [comment]
For ratings : schema.org have no tags for github specifically. 

As a placeholder
I have used github main page star as review count: 11300 and rating as 4.5.

File: index.html [hunk 1]
```diff
@@ -112,7 +112,285 @@
...
+      "creator": {
+          "@type": "Person",
+          "name": "Erik Bjare",
+          "url": "https://erik.bjareholt.com/",
+          "sameAs": "https://www.linkedin.com/in/erikbjareholt/"
+      },
+      "maintainer": [
+          {
+              "@type": "Person",
+              "name": "Erik Bjare",
```
Comment (ErikBjare) [inline index.html:296]:
```suggestion
              "name": "Erik Bjäreholt",
```

File: index.html [hunk 2]
```diff
@@ -112,7 +112,285 @@
...
+                "name": "ActivityWatch Firefox Extension",
+                "operatingSystem": "Firefox"
+              }
+            }
+          ]
+        },
+      "softwareRequirements": "Requires Windows 10 or later, macOS 10.15 or later, Linux kernel 5.4 or later, or Android 7.0 or later",
+      "creator": {
+          "@type": "Person",
+          "name": "Erik Bjare",
```
Comment (ErikBjare) [inline index.html:289]:
```suggestion
          "name": "Erik Bjäreholt",
```

File: index.html [hunk 3]
```diff
@@ -112,7 +112,285 @@
...
+      "applicationCategory": "https://en.wikipedia.org/wiki/Time-tracking_software",
+      "applicationSubCategory": ["DesktopApplication", "MobileApplication", "WebApplication"],
+      "aggregateRating": {
+          "@type": "AggregateRating",
+          "ratingValue": "4.5",
+          "reviewCount": "11300"
```
Comment (ErikBjare) [inline index.html:133]: Not really accurate to use number of stars as review count. Could instead use Google Play Store, or ProductHunt (both linked to on https://activitywatch.net/stats/).

File: index.html [hunk 4]
```diff
@@ -112,7 +112,285 @@
...
+              "@type": "Person",
+              "name": "Tymoteusz Czech",
+              "url": "https://github.com/Tymek"
+          },
+          {
+              "@type": "Person",
+              "name": "Glenn Willen",
+              "url": "https://github.com/gwillen"
+          }
+      ],
```
Comment (ErikBjare) [inline index.html:386]: These seem kinda random. If they are to be kept, they should prob reflect https://activitywatch.net/sponsors/

File: index.html [hunk 5]
```diff
@@ -112,7 +112,285 @@
...
+      "usageInfo": "https://docs.activitywatch.net/en/latest/getting-started.html?highlight=usage#usage",
```
Comment (ErikBjare) [inline index.html:387]:
```suggestion
      "usageInfo": "https://docs.activitywatch.net/en/latest/getting-started.html",
```

File: index.html [hunk 6]
```diff
@@ -112,7 +112,285 @@
...
+          "Editor plugins (IDE)",
+          "Local and privacy first",
+          "Cross-platform",
+          "Synchronization (We're working on it)"
+      ],
+      "audience": {
+          "@type": "Audience",
+          "audienceType": ["Business Professionals", "Freelancers", "Small Business", "ADHD individuals"]
+      },
+      "typicalAgeRange": "18-65",
```
Comment (ErikBjare) [inline index.html:156]: I'd like to think it's all ages :)

File: index.html [hunk 7]
```diff
@@ -112,7 +112,285 @@
...
+      "about": "ActivityWatch is a bundle of software that tracks your computer activity. You are, by default, the sole owner of your data. It also offers an ecosystem of software to work around it, including ways to collect more data and do different kinds of analysis. There are plenty of companies offering services which collect Quantified Self data with goals ranging from increasing personal productivity to understanding the people that managers manage (organizational productivity). However, all known services suffer from a significant disadvantage: the user’s data is in the hands of the service providers, which leads to a problem of trust. Every customer of these companies has their data in hands they are forced to trust if they want to use their service. This is a significant problem, but the true reason that we decided to do something about it was that existing solutions were inadequate. They focused on short-term insight, a goal worthy in itself, but we also want long-term understanding. We created ActivityWatch completely free and open source so anyone can use, improve and extend it.",
```
Comment (ErikBjare) [inline index.html:127]:
```suggestion
      "about": "ActivityWatch is a bundle of software that tracks your computer activity. You are, by default, the sole owner of your data. It also offers an ecosystem of software to work around it, including ways to collect more data and do different kinds of analysis. We created ActivityWatch completely free and open source so anyone can use, improve and extend it.",
```

**Review by ErikBjare — COMMENTED** [review]
Very nice! Just a few comments, then I'd be happy to merge it :)

**Comment by ErikBjare** [comment]
Oh, and you also cannot target gh-pages. You need to target the master branch from which the gh-pages branch is built.

**Comment by avadhut-the-merciful** [comment]
> Oh, and you also cannot target gh-pages. You need to include it the head template in the master branch (like https://github.com/ActivityWatch/activitywatch.github.io/blob/master/_includes/head.pug) from which the gh-pages branch is built.

 I'll make the changes and merge the branch into the master.

**Pull request closed without merging**